packages/components/tests/dummy/app/templates/components/link/standalone.hbs

```diff
@@ -144,65 +129,128 @@
         </li>
       </ol>
     </dd>
+    <dt>href</dt>
+    <dd>
+      <p>This is the URL that is passed down ("splatted") as arguments to the
+        <code>&lt;a&gt;</code>
+        element (see URL handling for details).</p>
+    </dd>
+    <dt>“splattributes”</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
   </dl>
 </section>
 
 <section>
-  <h3 class="dummy-h3">
-    Usage
-  </h3>
+  <h3 class="dummy-h3">Usage</h3>
 
-  <h4 class="dummy-h4">Basic invocation</h4>
+  <h4 class="dummy-h4">Basic use</h4>
+  <p class="dummy-paragraph">The most basic invocation requires both
+    <code class="dummy-code">icon</code>
+    and
+    <code class="dummy-code">text</code>
+    to be passed and an
+    <code class="dummy-code">href</code>
+    value (for details on how URLs are handled see below).</p>
   {{! prettier-ignore-start }}
   <CodeBlock
     @language="markup"
     @code='
-      <Hds::Link::Standalone @text="Watch tutorial video" @icon="film" href="/" />
+      <Hds::Link::Standalone @icon="film" @text="Watch tutorial video" href="..." />
     '
   />
   {{! prettier-ignore-end }}
   <p class="dummy-paragraph">Renders to:</p>
-  <Hds::Link::Standalone @text="Watch tutorial video" @icon="film" href="/" />
+  <Hds::Link::Standalone @icon="film" @text="Watch tutorial video" href="#" />
```

Comment:
This should have thrown an ember-template-lint error; is there a reason that the href value was changed? It should not be "blank" like this.

Reply:
> This should have thrown an ember-template-lint error
Nope, I didn't see errors.

> is there a reason that the href value was changed? It should not be "blank" like this.
Reason is that, if clicked by mistake while testing, it opens a new window with the index of the dummy website, and it's confusing (using the `#` at least remains on the same page, even if in a new window).